stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -615,8 +616,12 @@ internal class ChannelControllerImpl(
                 var attachment: Attachment = it
                 if (it.upload != null) {
                     val result = uploadAttachment(it, attachmentTransformer)
+                    attachment.extraData[EXTRA_UPLOAD_COMPLETE] = result.isSuccess
```

Comment:
What do you think if we expose for our domain this `uploadCompleted` as Boolean extensions field? And define set and get via this map with key `EXTRA_UPLOAD_COMPLETE `

Reply:
Done